### PR TITLE
Add docs status badge

### PR DESCRIPTION
**Pull request by novialriptide**

### CODE REVIEW

**Comment by Starbuck5** [comment]
I don't really see the point of adding this sort of badge. It doesn't even link to the docs.

**Comment by MyreMylar** [comment]
I guess we could make it link to the docs?

**Review by MyreMylar — CHANGES_REQUESTED** [review]
Make it link to the docs.

File: README.rst [hunk 1]
```diff
@@ -181,6 +182,8 @@ See docs/licenses for licenses of dependencies.
 .. |GithubCommits| image:: https://img.shields.io/github/commits-since/pygame-community/pygame-ce/2.1.3.svg
    :target: https://github.com/pygame-community/pygame-ce/compare/2.1.3...main
 
+.. |DocsStatus| image:: https://img.shields.io/website?down_message=offline&label=docs&up_message=online&url=https%3A%2F%2Fpyga.me%2Fdocs%2F
```
Comment (oddbookworm) [inline README.rst:185]: should this target the docs so that clicking on the docs badge takes you there?

**Review by oddbookworm — DISMISSED** [review]

File: README.rst [hunk 2]
```diff
@@ -181,6 +182,8 @@ See docs/licenses for licenses of dependencies.
...
+
```
Comment (MyreMylar) [inline README.rst:186]:
```suggestion
   :target: https://pyga.me/docs/
   
```

File: README.rst [hunk 3]
```diff
@@ -3,6 +3,7 @@
   :target: https://pyga.me/
 
 
+|AppVeyorBuild| |DocsStatus| |PyPiVersion| |PyPiLicense|
```
Comment (MyreMylar) [inline README.rst:6]:
```suggestion
|DocsStatus| |PyPiVersion| |PyPiLicense|
```

File: README.rst [hunk 4]
```diff
@@ -3,6 +3,7 @@
   :target: https://pyga.me/
 
 
+|DocsStatus| |PyPiVersion| |PyPiLicense|
```
Comment (MyreMylar) [inline README.rst:6]:
```suggestion
|DocsStatus| 
```

**Review by MyreMylar — APPROVED** [review]
LGTM 👍

**Review by ankith26 — APPROVED** [review]
Nice simple change, thanks! :cake:

**Pull request merged**